packer/nomad-server/scripts/nomad.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -x
+
+set -e
+
+files=/tmp/files
+
+sudo adduser \
+	--home=/var/lib/nomad \
+	--shell=/usr/sbin/nologin \
+	--system \
+	nomad
+
+mkdir -p /var/lib/nomad/
+mkdir -p /var/lib/nomad/log/
+
+sudo chown -R "nomad:nfsnobody" "/var/lib/nomad/"
+echo 1 > sudo tee /proc/sys/net/bridge/bridge-nf-call-arptables
+echo 1 > sudo tee /proc/sys/net/bridge/bridge-nf-call-ip6tables
+echo 1 > sudo tee /proc/sys/net/bridge/bridge-nf-call-iptables
+
+nomad_home=~nomad
```

Comment:
It would be good to define this first (`readonly NOMAD_HOME=/var/lib/nomad`), and then just refer to that everywhere else.